support/Tests/GraphQLSupportTests/IntegrationTests.swift

```diff
@@ -56,6 +56,24 @@ class IntegrationTests: XCTestCase {
         XCTAssertEqual(queryString, "mutation{set_integer(input:{key:\"answer\",value:42,negate:true})}")
     }
 
+    func testInputObjectExplictNilConstructor() {
+        let query = Generated.buildMutation { $0
+            .setInteger(input: Generated.SetIntegerInput(key: "answer", value: 42, negate: .some(nil)))
```

Comment:
I don't think so? I need a way to distinguish between an optional argument being set to it's default value (which is `nil`) and the user trying to tell me they want a property to be `nil`, hence the `.some(nil)`

Because it unwraps to nil, I can try to unwrap each of the optional arguments and then just use the value inside if it's there as the property's value